pkg/commands/drift/github.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package drift
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/abcxyz/guardian/pkg/github"
+)
+
+const (
+	issueTitle = "IAM drift detected"
+	issueBody  = `We've detected a drift between your submitted IAM policies and actual
+		IAM policies.
+
+		See the comment(s) below to see details of the drift
+
+		Please determine which parts are correct, and submit updated
+		terraform config and/or remove the extra policies.
+
+		Re-run drift detection manually once complete to verify all diffs are properly resolved.`
+)
+
+func createUpdateOrCloseIssues(ctx context.Context, token, owner, repo string, assignees, labels []string, drift *IAMDrift) error {
```

Comment:
As I predicted in my previous comment, this function is doing wayyy to much. We need to break it down into smaller pieces.

Reply:
Broke it out into two methods.